resilience4j-bulkhead/src/main/java/io/github/resilience4j/bulkhead/internal/ThreadPoolBulkheadAdapter.java

```diff
@@ -0,0 +1,66 @@
+package io.github.resilience4j.bulkhead.internal;
+
+import io.github.resilience4j.bulkhead.Bulkhead;
+import io.github.resilience4j.bulkhead.BulkheadConfig;
+import io.github.resilience4j.bulkhead.ThreadPoolBulkhead;
+
+import java.util.Map;
+import java.util.function.Supplier;
+
+public record ThreadPoolBulkheadAdapter(ThreadPoolBulkhead threadPoolBulkhead) implements Bulkhead {
```

Comment:
Could you please explain why this class/record is needed?

Reply:
I created that class as an adapter so that we didn't have to create a separate `ThreadPoolBulkheadAspectExt` interface and implementation and to avoid having to complicate the `PlainObjectBulkheadAspectExt` class by needing to handle both bulkhead types directly.